README.adoc

```diff
@@ -69,15 +68,31 @@ include::{common-includes}/gitclone.adoc[]
 [role='command']
 include::{common-includes}/twyb-intro.adoc[]
 
-Point your browser to the http://localhost:9080/openapi/ui[http://localhost:9080/openapi/ui^] URL. You see the OpenAPI
-user interface documenting the REST endpoints used in this guide. If you are interested
-in learning more about OpenAPI, read https://openliberty.io/guides/microprofile-openapi.html[Documenting RESTful APIs^]. 
-Expand the `/beanvalidation/validatespacecraft POST request to validate your
-spacecraft bean` section and click *Try it out*. Copy the following example input
-into the text box:
 
-[source, role='command']
-----
+// Static guide instruction
+ifndef::cloud-hosted[]
+Point your browser to the http://localhost:9080/openapi/ui[http://localhost:9080/openapi/ui^] URL.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Once your application is up and running, open another command-line session by selecting **Terminal** > **New Terminal** 
+from the menu of the IDE. Then use the following command to get the URL.
+Open your browser and check out your service by going to the URL that the command returns.
+```
+echo http://${USERNAME}-9080.$(echo $TOOL_DOMAIN | sed 's/\.labs\./.proxy./g')/openapi/ui
+```
+{: codeblock} 
+endif::[]
+
+You see the OpenAPI user interface documenting the REST endpoints used in this guide.
+If you are interested in learning more about OpenAPI,
+read https://openliberty.io/guides/microprofile-openapi.html[Documenting RESTful APIs^]. 
+Expand the `/beanvalidation/validatespacecraft POST request to validate your spacecraft bean` section 
```

Comment:
Expand the `/beanvalidation/validatespacecraft POST request to validate your spacecraft bean` section 
--->
Expand the `/beanvalidation/validatespacecraft` POST request to validate your spacecraft bean section

Reply:
`/beanvalidation/validatespacecraft POST request to validate your spacecraft bean` really is a piece of descriotion for the section

Reply:
ah ok thanks for clarifying